quickstarts/manage-data/airflow-airbyte-dbt.md

```diff
@@ -49,38 +48,32 @@ Open the code in your preferred IDE.
 Follow the instructions from [Getting started with ClearScape Analytics Experience](https://developers.teradata.com/quickstarts/get-access-to-vantage/clearscape-analytics-experience/getting-started-with-csae/) and acquire following:
 
 *  Host
-* username
-* password
+* Username
+* Password
 
 Jot it down, it will be needed in next step.
 
-## Edit `profiles.yml` file
-Navigate to `dbt_project` folder, open `profiles.yml` file, it contains configuration for dbt to connect with your data platform. Update this file with your Teradata connection details:
-* Host (from previous step)
-* username (from previous step)
-* password (from previous step)
-* schema: `airbyte_td`
-
-And move the file in location shown below:
-
-<ProfileTabs/>
-
 ## Airbyte setup
 
-1. After you install Airbyte OSS locally using `abctl`:
+1. After you install Airbyte OSS locally using `abctl`, generate and copy the credentials:
 
 ```bash
-abctl local install
+abctl local credentials
 ```
 
-2. Generate and copy the credentials:
+:::note 
+If you encounter `429 Too Many Requests` error with `abctl local install` command, check Docker and run it again.
+:::
```

Comment:
```suggestion
:::note 
At the moment of writing this quickstart, Airbyte installed with `abctl` performs unauthenticated pulls even if you pass in Docker credentials: https://github.com/airbytehq/airbyte/issues/46309. Since Docker limits unauthenticated pulls you may encounter `429 Too Many Requests` error. If you run into this error, launch Airbyte using `run-ab-platform.sh` scruot as described in [the Airbyte quickstart](https://developers.teradata.com/quickstarts/manage-data/use-airbyte-to-load-data-from-external-sources-to-teradata-vantage/#airbyte-open-source).
:::
```

Reply:
Thank you